### ISSUE TITLE
Transparency issue

### ISSUE THREAD

**Issue by narbartor** [shankarsivarajan/XNALaraMesh#1]
This was something that was sort of an issue in the original xnalara importer.

Back then some model transparency issue came from the fact that the viewport display blender mode was set to Alpha Blend which caused some models to look like this
![image](https://github.com/user-attachments/assets/77761410-786e-40fb-977a-b69fa2d74c56)

However that was something that could be solved in the original by changing the line of code where you could just change the line of code

materialData.blend_method = 'BLEND' into 'CLIP' instead

However due to how blender does shaders now, the options are a little different.

For 4.2 the way to solve the issue would be to change the Render method to "Dithered" or turn off transparency overlap
![image](https://github.com/user-attachments/assets/e73630b2-8bc4-4f74-820b-fa732659457d)

What Im asking for is basically, is there a way to make it default in the code so that I wouldnt have to manually do this for each object

**Comment by shankarsivarajan** [comment]
Yes, there is! You can replace the line (in `material_creator.py`) with
```
materialData.surface_render_method = 'DITHERED'
materialData.use_transparency_overlap = False
```
instead. See [API documentation](https://docs.blender.org/api/current/bpy.types.Material.html).

The docs say that's the default though, and it is for me. Did you perhaps change the setting in your Startup file?
